src/Middleware/OutputCaching/src/OutputCacheOptions.cs

```diff
@@ -45,8 +43,7 @@ public class OutputCacheOptions
     /// <summary>
     /// For testing purposes only.
     /// </summary>
-    [EditorBrowsable(EditorBrowsableState.Never)]
-    internal ISystemClock SystemClock { get; set; } = new SystemClock();
+    internal TimeProvider Time { get; set; } = TimeProvider.System;
```

Comment:
Why not call it `TimeProvider`?

Reply:
Agree, here it makes sense to call it TimeProvider. Auth is inconsistent but at least here it's pretty consistent (even SystemClock was called SystemClock).